graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/GqlAdminQuery.java

```diff
@@ -1,12 +1,23 @@
 package org.jahia.modules.graphql.provider.dxm.node;
 
 import graphql.annotations.annotationTypes.GraphQLDescription;
+import graphql.annotations.annotationTypes.GraphQLField;
 import graphql.annotations.annotationTypes.GraphQLName;
+import graphql.annotations.annotationTypes.GraphQLNonNull;
+import org.jahia.bin.Jahia;
 
 /**
  * GraphQL root object for Admin related queries.
  */
 @GraphQLName("adminQuery")
 @GraphQLDescription("Admin queries root")
 public class GqlAdminQuery {
+
+    @GraphQLField
+    @GraphQLName("version")
+    @GraphQLNonNull
+    @GraphQLDescription("Stub query which displays current Jahia product version")
```

Comment:
Just `Version of the running Jahia instance`, not need to say that's a "stub" we did for other reasons